lib/config.js

```diff
@@ -147,6 +147,15 @@ if (! process.env.NODE_ENV) {
   process.env.NODE_ENV = conf.get('env');
 }
 
+if(conf.get('env')  === 'development'){
```

Comment:
see 2 comments for this from https://github.com/mozilla/fxa-auth-server/pull/2369#pullrequestreview-107829584